test/e2e/plain-v0_provisioner_test.go

```diff
@@ -203,7 +205,7 @@ var _ = Describe("plain-v0 provisioner", func() {
 			Eventually(func() bool {
 				pod := &corev1.Pod{}
 				if err := c.Get(ctx, types.NamespacedName{
-					Name:      fmt.Sprintf("plain-unpack-bundle-%s", bundle.GetName()),
+					Name:      util.PodName("plain", bundle.GetName()),
```

Comment:
Note: it looks like this was potentially missed in the plain/k8s -> plain-v0 renaming PR as we hardcode the provisioners name in a couple of places. Right now the plain/v0 provisioner creates unpack Pods that are prefixed with `plain-unpack-bundle-*` vs. `plain-v0-unpack-bundle-*`.

Reply:
Yes, good call. I will open a small PR to address that issue.